app/embeds/index.js

```diff
@@ -148,6 +149,13 @@ export default [
     component: GoogleSlides,
     matcher: matcher(GoogleSlides),
   },
+  {
+    title: "Google Calendar",
+    keywords: "calendar",
+    icon: () => <Img src="/images/google-calendar.png" />,
```

Comment:
The icon needs to go all the way to the edge of the image or it will appear the wrong size in the UI, see the existing icons as examples:

https://github.com/outline/outline/blob/main/public/images/google-docs.png